### ISSUE TITLE
Empty id attribute in formgroup

### ISSUE THREAD

**Issue by akkie** [adrianhurt/play-bootstrap3#23]
Hi,

if I use a button as example:
```scala
@b3.submit('class -> "btn btn-primary btn-block", 'id -> "search"){@page("search")}
```

then this produces a form group with an empty ID:
```html
<div class="form-group  " id="">
	<button type="submit" class="btn btn-primary btn-block" id="search">Suchen</button>
</div>
```

This problem occurs also with compounded fields (date range). Normally empty id attributes shouldn't be added, because this leads to validation errors.

Best regards,
Christian

**Comment by adrianhurt** [comment]
Thank you @akkie. For the issue and specially for Silhouette! ;)

I've just fixed it in new 0.4.1-SNAPSHOT, or I hope so… Now the id is taken from `'_id` or `'id+_field` or it's hidden. So for your example it should take the value "search_field"

**Comment by akkie** [comment]
You are welcome! Works like a charm! Thanks
